src/utils/string-utils.js

```diff
@@ -5,5 +5,8 @@ export const formatPrice = number => {
   }).format(number);
 };
 
+export const emailValidator =
```

Comment:
Nice!
I would turn this into a util function to save you code where you implement it. 

```
const emailValidator = (email) => {
  const emailValidatingRegex = "/^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/"
  return email.test(emailValidationRegex)
}
```
And then implementation looks like:
```
const isEmailValid = emailValidator(email)
```